File: lib_video/src/main/java/com/effective/android/video/ui/cover/ICoverReceivingLayer.java
package com.effective.android.video.ui.cover;

/**
 *  浮层ui层变化
 *  Created by g8931 on 2019/04/20.
 */
public interface ICoverReceivingLayer {

    void onCoverClick();

    void onControlVisible(boolean visible);

    boolean onBack();

    boolean onFullScreen(boolean unfold);
}

File: lib_video/src/main/java/com/effective/android/video/ui/cover/ICoverReceivingLayer.kt
package com.effective.android.video.ui.cover

/**
 * 浮层ui层变化
 * Created by yummylau on 2019/04/20.
 */
interface ICoverReceivingLayer {

    fun onCoverClick()

    fun onControlVisible(visible: Boolean)

    fun onBack(): Boolean

    fun onFullScreen(unfold: Boolean): Boolean
}
